src/GraphQL/Validation/Rules.Custom/InputFieldsAndArgumentsOfCorrectLength.cs

```diff
@@ -81,7 +81,7 @@ private static void CheckLength(IHaveValue node, IProvideMetadata? provider, Val
             {
                 CheckStringLength(strLiteral.Value);
             }
-            else if (node.Value is VariableReference vRef && context.Variables != null)
```

Comment:
OK ? `context.Variables` property is initialized with `Inputs.Empty` now and non nullable so I removed this check.